guides/dialogflow/fulfillment/fulfillment-unavailability-hanlder.md

```diff
@@ -0,0 +1,29 @@
+# Dialogflow Fulfillment Unavailability Handler
+
+Sometimes conversational journeys require some response from fulfillment service. For example: in the end of ID&V process fulfillment service should return some [Followup Event Input](https://cloud.google.com/dialogflow/es/docs/fulfillment-webhook#event), but fulfillment service hasn't responded within 5(by default) seconds for some reasons. In this case Dialogflow will respond with the default text response which breaks normal flow as fulfillment service couldn't confirm result of the ID&V process. That's why it's important to escalate calls like that to some "failover" endpoint.
```

Comment:
...as fulfillment service `can't` confirm
...to escalate `such calls` to some...